ros/src/actuation/vehicles/packages/as/CMakeLists.txt

```diff
@@ -1,40 +1,49 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(as)
 
-# If submodule is not cloned this project will not be build.
-set(AS_MSG_PATH "${CMAKE_SOURCE_DIR}/msgs/platform_automation_msgs/module_comm_msgs")
-if(EXISTS "${AS_MSG_PATH}")
```

Comment:
Moving this IF statement always makes catkin search for module_comm_msgs.
So, I got error:
```
-- Could not find the required component 'module_comm_msgs'. The following CMake error indicates that you either need to install the package with the same name or change your environment so that it can be found.
CMake Error at /opt/ros/kinetic/share/catkin/cmake/catkinConfig.cmake:83 (find_package):
  Could not find a package configuration file provided by "module_comm_msgs"
  with any of the following names:

    module_comm_msgsConfig.cmake
    module_comm_msgs-config.cmake

  Add the installation prefix of "module_comm_msgs" to CMAKE_PREFIX_PATH or
  set "module_comm_msgs_DIR" to a directory containing one of the above
  files.  If "module_comm_msgs" provides a separate development package or
  SDK, be sure it has been installed.
Call Stack (most recent call first):
  actuation/vehicles/packages/as/CMakeLists.txt:4 (find_package)
```

Reply:
I know to need `git submodule update --init --recursive`
But, I think that autoware does not necessarily need to compile "as" package because problems often occur frequently by not executing this command.

If you want to use "as" default, I think that you need to include this command in catkin_make_release, install manual, or etc.
How about you think?

Reply:
Then if this package is not needed I recommend to add a CATKIN_IGNORE

Reply:
:+1: